spec/lib/va_profile/models/v3/validation_address_spec.rb

```diff
@@ -45,7 +45,6 @@
           'country' => {
```

Comment:
country_code_iso2 was removed because the AddressValidationController does not permit country_code_iso2